express_relay/sdk/js/src/types.d.ts

```diff
@@ -0,0 +1,319 @@
+/**
+ * This file was auto-generated by openapi-typescript.
+ * Do not make direct changes to the file.
+ */
```

Comment:
how did you generate this file? Can you put instructions for how to regenerate this (and when it's necessary) into the README?

Reply:
It may also be worthwhile to do the same thing for the ABI. Use `cast inspect abi` on the contract and import that as a json file.